Makefile.in

```diff
@@ -347,6 +347,8 @@ runtest-upstream:
 	cp $(stage2_prefix)/lib/ocaml/compiler-libs/*.cma _runtest/compilerlibs
 	cp $(stage2_prefix)/lib/ocaml/compiler-libs/*.a _runtest/compilerlibs
 	cp $(stage2_prefix)/lib/ocaml/compiler-libs/*.cmxa _runtest/compilerlibs
+	cp _build2/default/backend/debug/dwarf/dwarf_flags/dwarf_flags.cma \
```

Comment:
Why are the new files not installed in compiler-libs as usual (and then this like can use $(stage2_prefix) like the rest of the code around it)?

Reply:
Fixed